targets/XPlatCppSdk/source/code/source/playfab/PlayFabCurlHttpPlugin.cpp

```diff
@@ -99,6 +106,55 @@ namespace PlayFab
         } // UNLOCK httpRequestMutex
     }
 
+    // TODO: This should be merged with ExecuteRequest below
```

Comment:
I know these should be better merged, but I'm running low on time and wanted to change as little as possible about the original PlayFab SDK. I'll add a task in VSO for this one and the WinHttp one so we don't forget about it